tests/qepsi4/_ccsd_amplitude_parser_test.py

```diff
@@ -0,0 +1,290 @@
+import pytest
+from qepsi4 import run_psi4, parse_amplitudes_from_psi4_ccsd
+from openfermion import FermionOperator
+
+
+def get_expected_mp2_fop():
+    result_fop = FermionOperator()
+    result_fop += -0.0193104873 * FermionOperator("4^ 0 5^ 1")
```

Comment:
I assume that we cannot create a test for an example that is simple enough that we could calculate it on paper? Cause it looks to me that these are just generated by the code you're trying to test, so they don't really check whether the code works properly, it just checks whether the code does not change.